lib/commons/dom/get-tabbable-elements.js

```diff
@@ -12,12 +13,7 @@ function getTabbableElements(virtualNode) {
   const nodeAndDescendents = querySelectorAll(virtualNode, '*');
 
   const tabbableElements = nodeAndDescendents.filter(vNode => {
-    const isFocusable = vNode.isFocusable;
```

Comment:
We should continue to use the cached version on vNode. Best to revert this one too.